### PR TITLE
Update sparkasse and comdirect

### PR DESCRIPTION
**Pull request by MartinThoma**

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -285,7 +285,9 @@ websites:
       twitter: comdirect
       facebook: comdirect
       img: comdirect.png
-      tfa: No
+      tfa: Yes
+      software: Yes
+      doc: https://www.comdirect.de/konto/photo-tan.html#Vorteile
       lang: de
```
Comment (kmpoppe) [inline _data/banking.yml]:
```suggestion
```

File: _data/banking.yml [hunk 2]
```diff
@@ -995,7 +997,9 @@ websites:
       facebook: sparkasse
       email_address: sparkasse@sparkassen-finanzportal.de
       img: sparkasse.png
-      tfa: No
+      tfa: Yes
+      hardware: Yes
+      doc: https://www.sparkasse.de/service/sicherheit-im-internet/ihre-daten-bei-der-sparkasse.html
       lang: de
```
Comment (kmpoppe) [inline _data/banking.yml]:
```suggestion
```

File: _data/banking.yml (hunk 1 above)
Comment (kmpoppe) [reply _data/banking.yml]: As per rake: 'lang' should NOT be present when tfa: true.

File: _data/banking.yml (hunk 2 above)
Comment (kmpoppe) [reply _data/banking.yml]: As per rake: 'lang' should NOT be present when tfa: true.

**Review by kmpoppe — CHANGES_REQUESTED** [review]
See single line comments.

**Comment by kmpoppe** [comment]
Hey @MartinThoma - as you set the commit to be only changeable by you, I'd kindly ask you to remove the lang: tags because they are no longer needed, when tfa is already true. 
Thank you!
// Kai

**Comment by MartinThoma** [comment]
Ok, I removed them :+1:

**Comment by kmpoppe** [comment]
@MartinThoma Thank you for your contribution. I just now looked at the pages you provided, and even though they speak of "Zwei-Faktor-Authentifizierung" (two-factor authentication), what they really mean is authorization. Two Factor Authentication, as we see it for this project, means that you are directly challenged to provide a second factor upon login, not just for a sensitive transaction. This is also described in our Contribution Guidelines: https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#a-note-on-definitions

Therefore I think that this pull request can not be merged. If I perceived any information wrongly, please do not hesitate to tell me so. Otherwise, I'd be glad, if you closed this PR.

Thanks again for taking the time to contribute to our project!

// Kai

**Comment by MartinThoma** [comment]
> I just now looked at the pages you provided, and even though they speak of "Zwei-Faktor-Authentifizierung" (two-factor authentication), what they really mean is authorization. Two Factor Authentication, as we see it for this project, means that you are directly challenged to provide a second factor upon login, not just for a sensitive transaction.

Ok, I didn't know that. In this case you are right, the PR should not be merged. I close it.

**Pull request closed without merging**